Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -286,7 +286,7 @@ def get_threats_request(self, content_hash=None, mitigation_status=None, created
             siteIds=site_ids,
             rank=int(rank) if rank else None,
             keys_to_ignore=keys_to_ignore,
-            incidentStatuses=argToList(incident_statuses.lower() if incident_statuses is not None else None)
+            incidentStatuses=incident_statuses.lower() if incident_statuses is not None else None
```

Comment:
```suggestion
            incidentStatuses=incident_statuses.lower() if incident_statuses else None
```
wouldn't this be the same statement? don't you think?

Reply:
yes will make the change

Reply:
done